src/styles/blocks/card.scss

```diff
@@ -0,0 +1,156 @@
+.cards {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  place-content: center;
+  place-items: center;
+}
+
+@media (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, $card-width);
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+  transition: transform 300ms ease;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

Comment:
The usage of fixed values for margins (32px 19px 40px) can cause issues with responsiveness. Consider using relative units like em or percentages, or define these values as variables if they are used consistently throughout the project.